packages/ui/src/containers/TermsOfUse/index.tsx

```diff
@@ -1,31 +1,54 @@
-import TermsOfUseComponent from '@/components/TermsOfUse';
-import usePlatform from '@/hooks/use-platform';
+import classNames from 'classnames';
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Checkbox from '@/components/Checkbox';
+import TermsLinks from '@/components/TermsLinks';
 import useTerms from '@/hooks/use-terms';
+import { onKeyDownHandler } from '@/utils/a11y';
+
+import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
 };
 
 const TermsOfUse = ({ className }: Props) => {
```

Comment:
nit: rename this component to something generic like "LegalDocs" to distinguish from the specific "terms of use"

Reply:
updated as TermsAndPrivacy